sbi/neural_nets/estimators/base.py

```diff
@@ -224,11 +224,12 @@ def sample_and_log_prob(
 
 
 class ConditionalVectorFieldEstimator(ConditionalEstimator):
-    r"""Base class for vector field (e.g., score and ODE flow) estimators.
+    r"""Base class for conditional vector field (e.g., score and ODE flow) estimators.
 
-    The vector field estimator class is a wrapper around neural networks that allows to
-    evaluate the `vector_field`, and provide the `loss` of $\theta,x$ pairs. Here
-    $\theta$ would be the `input` and $x$ would be the `condition`.
+    The conditional vector field estimator class is a wrapper around neural
+    networks that allows to evaluate the `vector_field`, and provide the `loss`
+    of $\theta,x$ pairs. Here $\theta$ would be the `input` and $x$ would be
+    the `condition`.
```

Comment:
this has the potential to create merge conflicts down the line because it is not directly related to this PR and others are working on VF estimators.